app/views/report/_reports.haml

```diff
@@ -0,0 +1,42 @@
+.col-md-9
+  #reports
+    %h1
+      = t("report.title")
+    - if @reports.size == 0
+      %p
+        = t("report.unreviewed_reports", count: 0)
+    - @reports.each do |report|
+      - if report.item
+        .panel.panel-default
+          - username = report.user.username
+          .panel-heading
+            .reporter.pull-right
+              = raw t("report.reported_label", person: link_to(username, user_profile_path(username)))
```

Comment:
Tagging a string as html safe may be a security risk, prefer `safe_join` or other Rails tag helpers instead.